yarn-project/archiver/src/archiver/config.ts

```diff
@@ -22,6 +22,11 @@ export interface ArchiverConfig {
    */
   apiKey?: string;
 
+  /**
+   * The chain's ID
+   */
+  chainId?: number;
```

Comment:
Can we just name this l1ChainId (assuming that's what it represents). Also the same for the env var.